lib/addons/p5.dom.js

```diff
@@ -308,6 +308,87 @@
   };
 
   /**
+   * Creates a checkbox &lt;input&gt;&lt;/input&gt; element in the DOM.
+   *
+   * @method createCheckbox
+   * @param  {String} [label] label displayed after checkbox
+   * @param  {boolean} [value] value of the checkbox; checked is true, unchecked is false
+   * @return {Object/p5.Element} pointer to p5.Element holding created
```

Comment:
ah i didn't realize that, i was looking at another place in a core function. that'd be great, thanks!